@navikt/core/react/src/form/combobox/Input/InputController.tsx

```diff
@@ -88,10 +94,9 @@ export const InputController = forwardRef<
             onClick={clearInput}
             className="navds-combobox__button-clear"
             tabIndex={-1}
+            aria-hidden
```

Comment:
aria-hidden on button results in this error message in the console:

> Blocked aria-hidden on an element because its descendant retained focus. The focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden.
Element with focus: button

Wondering if I should use a div instead? 🤔

Reply:
Since both this and `ToggleListButton` are both aria-hidden, can the wrapper-`div` get aria-hidden to solve this?
<img width="268" alt="Screenshot 2024-12-02 at 09 27 43" src="https://github.com/user-attachments/assets/f0cebdd8-5d5c-4d81-9341-54b52eef8e37">

Reply:
Doesn't help

Reply:
Is there actually a reason to hide this element? In Search we opt to not hide it, so is there an actual difference in expected UX here?